### PR TITLE
feat(proto): add cosmos Tx and celestia MsgPayForBlobs

### PR DESCRIPTION
**Pull request by zvolin**

### CODE REVIEW

File: proto/build.rs [hunk 1]
```diff
@@ -86,12 +87,15 @@ fn main() -> Result<()> {
...
                 "vendor/celestia/da/data_availability_header.proto",
+                "vendor/celestia/blob/v1/tx.proto",
                 "vendor/header/pb/extended_header.proto",
                 "vendor/share/p2p/shrexnd/pb/share.proto",
                 "vendor/share/eds/byzantine/pb/share.proto",
                 "vendor/cosmos/base/v1beta1/coin.proto",
                 "vendor/cosmos/base/abci/v1beta1/abci.proto",
+                "vendor/cosmos/crypto/multisig/v1beta1/multisig.proto",
                 "vendor/cosmos/staking/v1beta1/query.proto",
+                "vendor/cosmos/tx/v1beta1/tx.proto",
```
Comment (fl0rek) [inline proto/build.rs:98]: to confirm, these are the new definitions we need and the rest are protobuf imported dependencies?

**Review by fl0rek — APPROVED** [review]
protobuf :rocket:

**Review by oblique — APPROVED** [review]

File: proto/build.rs (hunk 1 above)
Comment (oblique) [reply proto/build.rs:98]: Previously we vendored partially Cosmos protobuf. For this PR we need `MsgPayForBlobs` and it's dependencies.

Comment (zvolin) [reply proto/build.rs:98]: yeah, from cosmos we need the `Tx` and from celestia we need `MsgPayForBlobs`

**Pull request merged**